rllib/models/preprocessors.py

```diff
@@ -353,8 +353,23 @@ def get_preprocessor(space: gym.Space) -> type:
     if isinstance(space, (gym.spaces.Discrete, gym.spaces.MultiDiscrete)):
         preprocessor = OneHotPreprocessor
     elif obs_shape == ATARI_OBS_SHAPE:
```

Comment:
can we instead enable the atari pre processors when we are using atari environments in tests. 

Most people aren't using these by default so we probably don't need to enable these by default